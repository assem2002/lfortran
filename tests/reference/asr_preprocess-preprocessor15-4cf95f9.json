{
    "basename": "asr_preprocess-preprocessor15-4cf95f9",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor15.f90",
    "infile_hash": "b47691d49753905026191cfe1a1aa17ab0a7652f8b0aea9e1b1639e5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor15-4cf95f9.stdout",
    "stdout_hash": "7b65f42748045e46f8520245721d0f8670a95da99aca92efb9529a11",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}