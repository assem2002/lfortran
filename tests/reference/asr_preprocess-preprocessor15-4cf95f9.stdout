(TranslationUnit
    (SymbolTable
        1
        {
            preprocessor15:
                (Program
                    (SymbolTable
                        2
                        {
                            
                        })
                    preprocessor15
                    []
                    [(Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 4"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 4"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 4"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 2"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 2"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 2"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 1"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 1"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 1"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 2"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 2"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )
                    (Print
                        [(StringFormat
                            ()
                            [(StringConstant
                                "X == 2"
                                (Character 1 6 ())
                            )]
                            FormatFortran
                            (Character -1 0 ())
                            ()
                        )]
                        ()
                        ()
                    )]
                )
        })
    []
)
